### ISSUE TITLE
Leading whitespace ignored

### ISSUE THREAD

**Issue by jeffijoe** [formatjs/intl-messageformat#114]
Example:

```
Booked {numAttendees}{numMaxAttendees, plural,
      =0 {}
      other { / #}
   }
```

Results in

> Booked 2/ 3

When it should have been

> Booked 2 / 3

Is this correct behavior? If it is, that would suck.

**Comment by micaste** [comment]
I have the same problem in version 1.1.0. It is unproper and I hope the bug will fixed but you can work around it using a non-breakable space in your template : 
```
Booked {numAttendees}{numMaxAttendees, plural,
      =0 {}
      other {\u00A0/ #}
   }
```

**Comment by rightaway** [comment]
+1

**Comment by longlho** [comment]
fixed in 94abd759b04e740a772b825bffdfdef98c7dc7d8
